server/base/src/main/java/org/apache/accumulo/server/util/ListVolumesUsed.java

```diff
@@ -111,42 +112,28 @@ private static void listTable(String name, ServerContext context) throws Excepti
 
     volumes.clear();
 
-    scanner.clearColumns();
-    scanner.setRange(MetadataSchema.DeletesSection.getRange());
-
-    for (Entry<Key,Value> entry : scanner) {
-      String delPath = entry.getKey().getRow().toString()
-          .substring(MetadataSchema.DeletesSection.getRowPrefix().length());
-      volumes.add(getTableURI(delPath));
-    }
-
     System.out.println("Listing volumes referenced in " + name
         + " deletes section (volume replacement occurrs at deletion time)");
 
-    for (String volume : volumes) {
-      System.out.println("\tVolume : " + volume);
+    Iterator<String> delPaths = context.getAmple().getGcCandidates(name, "");
+    while (delPaths.hasNext()) {
+      System.out.println("\tVolume : " + getTableURI(delPaths.next()));
```

Comment:
This should probably add to the `volumes` in the loop (and print the set after the loop) like the old code did inorder to only print the unique set of volumes.